### PR TITLE
Support pprof

### LINKED ISSUE #238: Enable pprof in all conponents of Chaos Mesh
## Feature Request

**Is your feature request related to a problem? Please describe:**
 
Sometimes we need to locate some Chaos Mesh issue, such as performance, memory leaks, etc. I think `pprof` can help us. So I recommend enabling `pprof` in all components of chaos-mesh

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

### What problem does this PR solve? 

Fixes #238

### What is changed and how does it work?

For chaos-damon, just registering pprof handler to the http sever.
But for controller-manager, because https://github.com/kubernetes-sigs/kubebuilder/issues/1225, it would be not that clear to use the same metrics port in pprof. So I add a new pprof port, which can be disabled by setting "0"

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: cmd/controller-manager/main.go [hunk 1]
```diff
@@ -65,6 +68,7 @@ func init() {
 
 func parseFlags() {
 	flag.StringVar(&metricsAddr, "metrics-addr", ":10080", "The address the metric endpoint binds to.")
+	flag.StringVar(&pprofAddr, "pprof-addr", "0", "The address the pprof endpoint binds to.")
```
Comment (zhouqiang-cl) [inline cmd/controller-manager/main.go]: I think we should enable pprof default

File: helm/chaos-mesh/values.yaml [hunk 2]
```diff
@@ -24,6 +24,9 @@ controllerManager:
   service:
     type: ClusterIP
 
+  # enableProfiling is a flag to enable pprof in controller-manager
+  enableProfiling: false
```
Comment (Yisaer) [inline helm/chaos-mesh/values.yaml]: I tend to make `enableProfiling` as a global switch to control to pprof both for `manager` and daemon`.

Comment (yeya24) [reply helm/chaos-mesh/values.yaml]: Thanks for your review! Updated.

**Pull request merged**